cmd/zed/zed.d/all-syslog.sh

```diff
@@ -6,6 +6,7 @@
 . "${ZED_ZEDLET_DIR}/zed-functions.sh"
 
 zed_log_msg "eid=${ZEVENT_EID}" "class=${ZEVENT_SUBCLASS}" \
-    "${ZEVENT_POOL:+"pool=${ZEVENT_POOL}"}" \
+    "${ZEVENT_POOL_GUID:+"pool_guid=${ZEVENT_POOL_GUID}"}" \
```

Comment:
It may be useful to log both here if the reason for the change is to minimize ambiguity.